test/e2e/node/pre_stop.go

```diff
@@ -169,4 +178,111 @@ var _ = SIGDescribe("PreStop", func() {
 	framework.ConformanceIt("should call prestop when killing a pod ", func() {
 		testPreStop(f.ClientSet, f.Namespace.Name)
 	})
+
+	It("graceful pod terminated should wait until preStop hook completes the process [Slow]", func() {
+		By("creating the pod")
+		name := "pod-prestop-hook-" + string(uuid.NewUUID())
+		value := strconv.Itoa(time.Now().Nanosecond())
+		pod := createPodWithpreStopLifeCycle(name, value)
```

Comment:
nit: `podWithPreStopLifecycleHook`

To me, the "create" verb implies interaction with the cluster, whereas here we're just instantiating a test fixture